lib/cmd/fh3/projects/list.js

```diff
@@ -13,21 +13,26 @@ module.exports = {
     cmd : 'fhc projects list --authorEmail=<authorEmail>',
     desc : "List of all projects with the <authorEmail>"
   },{
-    cmd : 'fhc projects list',
+    cmd : 'fhc projects list --json',
     desc : "List of all projects in json format"
+  }, {
+    cmd : 'fhc projects list --includeApps --json',
+    desc : "List of all projects in json format and include app info"
   }],
   'demand' : [],
   'alias' : {
     'authorEmail': 'a',
+    'includeApps': 'i',
     'json': 'j',
     0 : 'authorEmail'
   },
   'describe' : {
     'authorEmail' : i18n._("Email of the user author/owner of the project."),
     'json' : i18n._("Output in json format")
   },
-  'url': function() {
-    return 'box/api/projects';
+  'url': function(params) {
+    var includeApps = params.includeApps ? "true" : "false";
```

Comment:
Might params be undefined?

Reply:
No, even if you do a vanilla `fhc projects list` it will be defined.